aldryn_jobs/views.py

```diff
@@ -67,7 +67,9 @@ class JobOpeningDetail(AppConfigMixin, TranslatableSlugMixin, DetailView):
     form_class = JobApplicationForm
     template_name = 'aldryn_jobs/jobs_detail.html'
     slug_url_kwarg = 'job_opening_slug'
-    queryset = JobOpening.objects.all()
+
+    def get_queryset(self):
+        return JobOpening.objects.filter(category__app_config=self.config)
```

Comment:
https://github.com/aldryn/aldryn-jobs/blob/master/aldryn_jobs/managers.py#L33 or https://github.com/aldryn/aldryn-jobs/blob/master/aldryn_jobs/managers.py#L20
can be used...
which means you can use  `JobOpening.objects.namespace(self.namespace)` if self.namespace is being set correctly in `dispatch`